community/assessments/projects/k8gb/self-assessment.md

```diff
@@ -0,0 +1,234 @@
+# k8gb Self-assessment
+
+Security reviewers: Yury Tsarev, Jirka Kremser
+
+This document provides a self-assessment of the k8gb project following the guidelines outlined by the CNCF TAG Security group. The purpose is to evaluate k8gb’s current security posture and alignment with best practices, ensuring that it is suitable for adoption at a CNCF incubation level.
+
+## Table of Contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+  * [Intended use](#intended-use)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+  * [Future state](#future-state)
+* [Secure development practices](#secure-development-practices)
+  * [Deployment pipeline](#deployment-pipeline)
+  * [Communication channels](#communication-channels)
+* [Security issue resolution](#security-issue-resolution)
+  * [Responsible disclosure practice](#responsible-disclosure-practice)
+  * [Incident response](#incident-response)
+* [Appendix](#appendix)
+
+## Metadata
+
+### Security links
+|   |  |
+| - | - |
+| Software | https://github.com/k8gb-io/k8gb  |
+| Security Provider | No, k8gb is not a security provider. It relies on Kubernetes' built-in security mechanisms (e.g., RBAC, network policies) and does not implement standalone security features. |
+| Languages | Golang |
+| SBOM | https://github.com/k8gb-io/k8gb/releases/download/v0.14.0/k8gb_0.14.0_linux_amd64.tar.gz.sbom.json |
+| Security Insights | https://github.com/k8gb-io/k8gb/blob/master/SECURITY-INSIGHTS.yml |
+| Security File | https://github.com/k8gb-io/k8gb/blob/master/SECURITY.md |
+| Cosign pub-key | https://github.com/k8gb-io/k8gb/blob/master/cosign.pub |
+| | |
+
+### Intended use
+
+To increase the software supply chain security, we encourage our users to consume k8gb container images with Kyverno's admission webhook
+([/policy](https://kyverno.io/docs/writing-policies/verify-images/sigstore/#verifying-image-signatures)) that will ensure that
+images are signed and nobody had tampered with them. Our public key that can be used to verify this is in the root or our repository.
+
+## Overview
+
+k8gb is implemented using the Kubernetes operator pattern with a single CRD to enable Global Load Balancing. k8gb provides independent GSLB capability to any Ingress or Service without a dedicated management cluster, instead relying on timeproof DNS. k8gb has no single point of failure, and uses Kubernetes native application health checks (such as liveness and readiness probes) to update DNS to aid in load balancing decisions.
```

Comment:
need to define acronyms the first time they are used

Reply:
To be honest, I don't clearly understand what you're doing from this.  There are too many new terms and not enough context.  I read a fair amount of your docs and have a better understanding, but this should be clearer.